### ISSUE TITLE
Using terraform to run remote tests no longer works with terraform 0.7.0

### ISSUE THREAD

**Issue by arjunravinarayan** [cockroachdb/cockroach#8619]
When I run
```
make test \
         TESTTIMEOUT=48h \
         PKG=./acceptance \
         TESTS=Rebalance_3To5Small \
         TESTFLAGS='-v -remote -key-name google_compute_engine -cwd terraform -tf.keep-cluster=failed'
```
with terraform version 0.7.0, I get the following error:
```
+ terraform apply -var=num_instances=3 -var=stores=--store=/mnt/data0 -var=cockroach_disk_type=pd-standard -var=cockroach_env=COCKROACH_SKIP_UPDATE_CHECK=1 -no-color -var=key_name=google_compute_engine -state=rebal-3to5s-backstabbingpasteur.tfstate -var=prefix=rebal-3to5s-backstabbingpasteur
invalid value "stores=--store=/mnt/data0" for flag -var: Cannot parse value for variable stores ("stores=--store=/mnt/data0") as valid HCL: At 1:8: Unknown token: 1:8 SUB -
Usage: terraform apply [options] [DIR-OR-PLAN]
```

This make directive works fine with 0.6.16 (the last pre-0.7 version), so it must be that terraform has changed how they parse command line arguments.

**Comment by cuongdo** [comment]
Looks like this is a bug in Terraform 0.7.0 that'll be fixed in 0.7.1:

https://github.com/hashicorp/terraform/issues/7981

Related:

https://github.com/hashicorp/terraform/issues/7962

Workaround for now is to use 0.6.16.

**Comment by cuongdo** [comment]
Still broken as of Terraform 0.7.2, unfortunately!

**Comment by tamird** [comment]
Fixed in #9075.
